search.py

```diff
@@ -1,6 +1,10 @@
-
 import urllib
 import webbrowser
+from urllib.request import urlopen
+from bs4 import BeautifulSoup
```

Comment:
What are you trying to achieve with BeautifulSoup? It's probably useful but can you put this under a different pull request? Let's keep this one to just the test for now.